Sources/TuistAutomation/TuistAutomation.swift

```diff
@@ -1 +1 @@
-
+// TuistAutomation placeholder
```

Comment:
needed to add a placeholder as `swiftlint` doesn't like leading whitespace :) - this can be removed once some source code is added.

Alternatively we could remove the `TuistAutomation` module and re-add it once there some source code.